php/class-asset-manager-svg-sprite.php

```diff
@@ -110,7 +110,7 @@ public function get_global_attributes() {
 			 *     @type array $attribute Attribute name-value pairs.
 			 * }
 			 */
-			static::$_global_attributes = apply_filters( 'am_svg_attributes', [] );
+			static::$_global_attributes = apply_filters( 'am_global_svg_attributes', [] );
```

Comment:
Not sure how widely this filter is used, but you may want to add a deprecation for `am_svg_attributes` in case someone tries to upgrade and is using the old filter. You would do so by doing something like this...

```suggestion
			static::$_global_attributes = apply_filters_deprecated( 'am_svg_attributes', [ [] ], '1.1.0', 'am_global_svg_attributes' );
```

See: https://developer.wordpress.org/reference/functions/apply_filters_deprecated/

Reply:
Wow, thanks, good to know! This isn't in production yet, so not being used.